app/src/main/java/info/nightscout/android/medtronic/MainActivity.java

```diff
@@ -93,10 +94,24 @@
     private int chartZoom = 3;
     private boolean hasZoomedChart = false;
 
-    private NumberFormat sgvFormatter;
-    private boolean mmolxl;
+    public static NumberFormat sgvFormatter;
```

Comment:
Why has sgvFormatter to be static? Just not to recreate it after instance ist destroyed? There is no need to keep the formatter form my point of view.